fibernewrelic/fiber.go

```diff
@@ -66,33 +65,26 @@ func New(cfg Config) fiber.Handler {
 	}
 
 	return func(c *fiber.Ctx) error {
-		txn := app.StartTransaction("")
+		txn := app.StartTransaction(createTransactionName(c))
 		defer txn.End()
 
-		handlerErr := c.Next()
-
-		var (
```

Comment:
No need to do any copy for more clear implementation, I recommend to review it as `View File` section 😅